src/main/java/com/software/modsen/drivermicroservice/controllers/DriverController.java

```diff
@@ -0,0 +1,52 @@
+package com.software.modsen.drivermicroservice.controllers;
+
+import com.software.modsen.drivermicroservice.entities.driver.Driver;
+import com.software.modsen.drivermicroservice.entities.driver.DriverDto;
+import com.software.modsen.drivermicroservice.entities.driver.DriverPatchDto;
+import com.software.modsen.drivermicroservice.services.DriverService;
+import jakarta.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(value = "/api/driver", produces = "application/json")
+public class DriverController {
+    @Autowired
+    private DriverService driverService;
+
+    @GetMapping
+    public ResponseEntity<List<Driver>> getAllDriver() {
+        return ResponseEntity.ok(driverService.getAllDrivers());
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<Driver> getDriverById(@PathVariable("id") long id) {
+        return ResponseEntity.ok(driverService.getDriverById(id));
+    }
+
+    @PostMapping
+    public ResponseEntity<Driver> saveDriver(@Valid
+                                       @RequestBody DriverDto driverDto) {
+        return ResponseEntity.ok(driverService.saveDriver(driverDto));
```

Comment:
Minor: If you create resource, it will be better to use 201 (Created) status code